### PR TITLE
build: config eslint and add lint github action

### PR DESCRIPTION
**Pull request by codepushr**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm' # See documentation for possible values
+    directory: '/' # Location of package manifests
```
Comment (axi92) [inline .github/dependabot.yml:9]: Does this also check the sub dirs?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    schedule:
+      interval: 'monthly'
```
Comment (axi92) [inline .github/dependabot.yml:11]: Is monthly a better option as weekly?
I am just asking not judging, if monthly is the desired option we should make this change in the other repos too?!

**Review by axi92 — COMMENTED** [review]
I added some questions about the dependabot config, the rest looks good.

File: .github/dependabot.yml (hunk 2 above)
Comment (codepushr) [reply .github/dependabot.yml:11]: I've taken it from our react-native plugin, we can as well move to weekly if you like.

File: .github/dependabot.yml (hunk 1 above)
Comment (codepushr) [reply .github/dependabot.yml:9]: Not sure if it's desirable to have that level of alertness for a mere example-app. What are your thoughts on this?

Comment (axi92) [reply .github/dependabot.yml:9]: True! And vulnerabilities are tracked either way.

File: .github/dependabot.yml (hunk 2 above)
Comment (axi92) [reply .github/dependabot.yml:11]: Ok, I guess monthly is fine. So its just once a month the work and not every week.

**Review by axi92 — DISMISSED** [review]

**Review by axi92 — APPROVED** [review]

**Pull request merged**